compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -333,6 +333,10 @@ hir_analysis_start_not_target_feature = `#[start]` function is not allowed to ha
 hir_analysis_start_not_track_caller = `#[start]` function is not allowed to be `#[track_caller]`
     .label = `#[start]` function is not allowed to be `#[track_caller]`
 
+hir_analysis_static_mut_ref = use of mutable static is discouraged
```

Comment:
as this is for the error, we should use more strict wording than "discouraged": ideally we'd use something like "disallowed" or such.